content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -38,13 +37,20 @@ or alternatively, to bypass the Istio proxy for a specific range of IPs.
 
     Note that any pod that you can `exec` and `curl` from will do for the procedures below.
 
+*   Set the `SOURCE_POD` environment variable to the deployed `sleep` pod:
+
+    {{< text bash >}}
+    $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
+    {{< /text >}}
+
```

Comment:
If you are at it, should we replace executing into the pod by running `curl` directly, like `kubectl exec -it $SOURCE_POD -c sleep -- curl http://httpbin.org/headers` ?

In all the advanced tasks I use the latter way.

Reply:
Hmm, not a bad idea but I think I'll leave it as it is for now, since most of the basic tasks do it in 2 steps. It might be worth considering doing this for all tasks?

Reply:
Right, we may want to consider that for all the tasks.